lib/kafka/producer_manager.rb

```diff
@@ -59,6 +62,19 @@ def setup_instrumentation
 
         Rails.logger.info "WaterDrop [#{producer_id}] delivered message with offset: #{offset}"
       end
+
+      setup_datadog_instrumentation
+    end
+
+    def setup_datadog_instrumentation
+      listener = ::WaterDrop::Instrumentation::Vendors::Datadog::MetricsListener.new do |config|
+        statsd_addr = ENV['STATSD_ADDR'] || '127.0.0.1:8125'
```

Comment:
STATSD_ADDR in all environments is set to 127.0.0.1:8125 but better to use it in case that ever changes